### ISSUE TITLE
Fix NullPointer in Router.java when no routes

### ISSUE THREAD

**Issue by mitchdennett** [severell/severell#6]
Fix NullPointer in Router.java when no routes 

```
java.lang.NullPointerException
    at com.severell.core.http.Router.lookup(Router.java:136)
    at com.severell.core.http.Dispatcher.doRequest(Dispatcher.java:68)
```

Look at file Router.java line 136.

Please add unit test to cover this case.

**Comment by Belovedbb** [comment]
Can i hop on that?

**Comment by mitchdennett** [comment]
Fixed with #7
